File: src/main/java/ru/ioleksiv/telegram/bot/core/model/telegram/model/type/Venue.java
package ru.ioleksiv.telegram.bot.core.model.telegram.model.type;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * @link https://core.telegram.org/bots/api#venue
 */
public class Venue {
    @JsonProperty("location")
    private Location location;
    @JsonProperty("title")
    private String title;
    @JsonProperty("address")
    private String address;
    @JsonProperty("foursquare_id")
    private String foursquareId;

    public Location getLocation() {
        return location;
    }

    public void setLocation(Location location) {
        this.location = location;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getFoursquareId() {
        return foursquareId;
    }

    public void setFoursquareId(String foursquareId) {
        this.foursquareId = foursquareId;
    }
}

File: server-api/src/main/kotlin/io/github/oleksivio/tl/kbot/server/api/objects/std/Venue.kt
package io.github.oleksivio.tl.kbot.server.api.objects.std

import com.fasterxml.jackson.annotation.JsonProperty
import io.github.oleksivio.tl.kbot.server.api.model.ITelegram

/**
 *  [Venue](https://core.telegram.org/bots/api/#venue)
 */
data class Venue(
    /**
         * To setup filter:
         *
         *  LocationFilter location location Location Venue location
         */
        @JsonProperty("location")
        var location: Location? = null,
    /**
         * To setup filter:
         *
         *  StringFilter title title String Name of the venue
         */
        @JsonProperty("title")
        var title: String? = null,
    /**
         * To setup filter:
         *
         *  StringFilter address address String Address of the venue
         */
        @JsonProperty("address")
        var address: String? = null,
    /**
         * To setup filter:
         *
         *  NotNullFilter foursquareId foursquare_id [String] Optional. Foursquare identifier of the venue
         */
        @JsonProperty("foursquare_id")
        var foursquareId: String? = null,
    /**
         * To setup filter:
         *
         *  StringFilter foursquareType foursquare_type [String] Optional. Foursquare type of the venue.
         */
        @JsonProperty("foursquare_type")
        var foursquareType: String? = null
) : ITelegram
